capstone/capdb/models.py

```diff
@@ -744,8 +747,8 @@ def create_or_update_metadata(self, update_existing=True, save_self=True):
                     court = Court(**court_kwargs)
                     court.save()
                 case_metadata.court = court
-                if case_metadata.jurisdiction_id and court.jurisdiction_id != case_metadata.jurisdiction_id:
-                    court.jurisdiction_id = case_metadata.jurisdiction_id
+                if case_metadata.jurisdiction and court.jurisdiction != case_metadata.jurisdiction:
+                    court.jurisdiction = case_metadata.jurisdiction
                     court.save()
```

Comment:
This might be a huge edge case, but it strikes me as something possibly problematic (even before the changes in this pull request). If `case_metadata.jurisdiction_id` does not equal `court.jurisdiction_id` then should we change a court's jurisdiction based on what might be a print error?